src/PrestaShopBundle/Form/Admin/Configure/AdvancedParameters/Employee/EmployeeOptionsFormDataProvider.php

```diff
@@ -60,6 +71,33 @@ public function getData()
      */
     public function setData(array $data)
     {
+        if ($errors = $this->validate($data)) {
+            return $errors;
+        }
+
         return $this->employeeOptionsConfiguration->updateConfiguration($data);
     }
+
+    /**
+     * Perform validation on form data before saving it.
+     *
+     * @param array $data
+     *
+     * @return array Returns array of errors
+     */
+    protected function validate(array $data)
+    {
+        $errors = [];
+
+        $passwordChangeTime = $data['password_change_time'];
+        if (!is_numeric($passwordChangeTime) || 0 > $passwordChangeTime) {
+            $errors[] = [
+                'key' => 'The %s field is invalid.',
```

Comment:
```suggestion
                'key' => 'The field is invalid. Please enter a positive integer.',
```

Reply:
Wasn't this issue already fixed in https://github.com/PrestaShop/PrestaShop/pull/21234#issuecomment-729622193 ?

Reply:
@Julievrz no, the error message is displayed at the top of the page. I just copy code from another file, the translation `The %s field is invalid.` is used in same context. In my opinion, there is no need to create a new translation here.

Reply:
I think we should, on the contrary. Saying "The field is invalid" without explaining the problem to users can be very frustrating for them. :/ 

I think we can use "The %s field is invalid. Please enter a positive integer." so users know how to solve the issue. :)

Reply:
In this case, we need to use this translation on every files that handle an error of the same type.

Reply:
Ok, then I'm revoming the waiting for wording label. :)

Reply:
Thanks @Julievrz. I think we can make issue about not enough explicit error messages. Can you handle it?